### ISSUE TITLE
Error with BBB 2.3.0

### ISSUE THREAD

**Issue by vasylOstrovskyi** [mconf/bigbluebot#24]
Bigbluebot does not work with bbb 2.3.0. My steps:
1. git clone git clone https://github.com/mconf/bigbluebot.git
2. npm i bigbluebot. as a result, node/modules/bigbluebot is a link to the cloned from github version
3. create .env file as documented in the manual. Added 
```
BIGBLUEBOT_HOST=https://test.bigbluebutton.org
BIGBLUEBOT_BOTS=1

```
4. create run.js with
```
const bigbluebot = require('bigbluebot');

const actions = async page => {
  await bigbluebot.audio.modal.microphone(page);
  await bigbluebot.audio.mute(page);
};

bigbluebot.run(actions);

```
5. node run. js. All works fine, PASSed
6. Change url->version: 2.3 in bigbluebot/config/config.json (should be documented in README.md)
7. change `BIGBLUEBOT_HOST=https://test23.bigbluebutton.org` in .env (also tried my production server)
8. node run.js raises error:
```
INFO 	2021-05-03T13:41:11.970Z Bots: 1
INFO 	2021-05-03T13:41:11.971Z Life: 60 seconds
INFO 	2021-05-03T13:41:11.971Z Fetching uk_UA:en_GB:en locale
INFO 	2021-05-03T13:41:15.600Z Gunner Ankunding: join https://test23.bigbluebutton.org at Demo Meeting
ERROR 	2021-05-03T13:41:21.556Z Missing label undefined
ERROR 	2021-05-03T13:41:21.556Z TypeError: Cannot read property 'indexOf' of undefined
    at aria (/home/vo/bbb/bot23/bigbluebot/lib/util.js:64:25)
    at translate (/home/vo/bbb/bot23/bigbluebot/lib/util.js:114:20)
    at Object.join (/home/vo/bbb/bot23/bigbluebot/lib/util.js:146:22)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async /home/vo/bbb/bot23/bigbluebot/lib/run.js:49:22
    at async Promise.all (index 0)
    at async /home/vo/bbb/bot23/bigbluebot/lib/run.js:61:7
    at async Promise.all (index 0)
    at async Object.run (/home/vo/bbb/bot23/bigbluebot/lib/run.js:81:3)

```
Regards,
 Vasyl

**Comment by pedrobmarin** [comment]
@vasylOstrovskyi , although the bots do not have full support on 2.3 yet since some labels where changed for the audio interaction, this particular issue you can resolve by setting `2.3` at the bot config. https://github.com/mconf/bigbluebot/blob/0e27c5e7ca4de82314bf300a027d205f6fb4403e/config/config.json#L4

**Comment by pedrobmarin** [comment]
There's no full support for 2.3 right now (and that's why there is no documentation about using with 2.3)

**Comment by vasylOstrovskyi** [comment]
Thank you for quick reply.
Ok, as I mentioned in  (6), I did that. Some simple investigations indicate that something went wrong with obtaining locale name (as you can see from the log above, the program tried to get several locales), so I manually (just for test) changed line 240 in lib/util.js to `const lang = 'en';`, after that, I  have the following error
```
INFO 	2021-05-03T14:11:30.882Z Bots: 1
INFO 	2021-05-03T14:11:30.884Z Life: 60 seconds
INFO 	2021-05-03T14:11:30.884Z Creating meeting
INFO 	2021-05-03T14:11:31.082Z Fetching en locale
DEBUG 	2021-05-03T14:11:31.170Z Opening browser
INFO 	2021-05-03T14:11:33.224Z Alivia Kuhic: join https://bbb.imath.kiev.ua at 7cdomquxysuj7udhe90uedeugdn7p70q4u73liv3
ERROR 	2021-05-03T14:12:34.513Z TimeoutError: waiting for selector `[aria-label^="Options"]` failed: timeout 60000ms exceeded
    at new WaitTask (/home/vo/bbb/node_modules/puppeteer/lib/cjs/puppeteer/common/DOMWorld.js:509:34)
    at DOMWorld.waitForSelectorInPage (/home/vo/bbb/node_modules/puppeteer/lib/cjs/puppeteer/common/DOMWorld.js:420:26)
    at Object.internalHandler.waitFor (/home/vo/bbb/node_modules/puppeteer/lib/cjs/puppeteer/common/QueryHandler.js:31:77)
    at DOMWorld.waitForSelector (/home/vo/bbb/node_modules/puppeteer/lib/cjs/puppeteer/common/DOMWorld.js:313:29)
    at Frame.waitForSelector (/home/vo/bbb/node_modules/puppeteer/lib/cjs/puppeteer/common/FrameManager.js:841:51)
    at Page.waitForSelector (/home/vo/bbb/node_modules/puppeteer/lib/cjs/puppeteer/common/Page.js:1280:33)
    at Object.join (/home/vo/bbb/node_modules/bigbluebot/lib/util.js:147:16)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async /home/vo/bbb/node_modules/bigbluebot/lib/run.js:49:22
    at async Promise.all (index 0)

```
Probably, 2.3 is evolving too fast to follow all its labels and popups ;)
Regards,
 Vasyl.

**Comment by pedrobmarin** [comment]
Yes. As far as I know the audio interaction is the only big change. You won't be able to mute, unmute and leave audio with the bots unless you disable the dynamic audio feature.

**Comment by pedrobmarin** [comment]
But I had zero time to check on that so far... I'll try to do it on a next free weekend

**Comment by vasylOstrovskyi** [comment]
Oh, sorry, I manually changed locale to 'en' and tried with my server but my server accepts only 'uk_UA' by default.
With manually set to 'en', test23.bigbluebutton.org works fine, and with manually set to 'uk_UA' it works with my server.
However without setting locale explicitly (as shipped) the program does not recognize which locale to take.
Once again: by default, the program is looking for the following
INFO 	2021-05-03T13:41:11.971Z Fetching uk_UA:en_GB:en locale
Regards,
 Vasyl

**Comment by pedrobmarin** [comment]
Right! Thanks for the feedback. I'll have this in mind while trying to fix this up. I'll probably have to move all labels to use the `data-test` to avoid multiple problems the new localization introduced.

**Comment by pedrobmarin** [comment]
Hi @vasylOstrovskyi ,

I included some changes for the locales 744ee63514c25747b5ec9c07b40e996d839c32e3 . I think that it might have solved this issue.
